GTests/global_glass.cuh

```diff
@@ -0,0 +1,250 @@
+#include <cstdint>
```

Comment:
I think we need some wrapper kernel to call the functions from the host side but I bet we could do some generic one that takes in a function point with N arguments and then leverage that for multiple functions.

Reply:
i made the file because we can't call device functions in host functions, only global functions can do that. So like Professor Plancher said we need a wrapper